src/cryptojwt/key_bundle.py

```diff
@@ -365,6 +369,16 @@ def do_remote(self):
         # if self.verify_ssl is not None:
         #     self.httpc_params["verify"] = self.verify_ssl
 
+        if self.last_error:
+            t = self.last_error + self.error_holddown
```

Comment:
since we're always going to be doing this, we can do it once when setting `self.last_error` - and thus it should be called something like `ignore_errors_until` (which I also find more explainatory than "holddown" and "last_error").

```python
class KeyBundle():
    ...
    self.ignore_errors_period = ignore_errors_period
    self.ignore_errors_until = 0 # UNIX timestamp of last error
    ...

    def do_remote():
        ...
        if time.time() < self.ignore_errors_until:
            warn()
            return False
        ...
        if error:
            self.ignore_errors_until = time.time() + self.ignore_errors_period
            raise UpdateFailed()
        ...
        self.ignore_errors_until = 0
        return True
```

Reply:
Good point, fixed in 6ac7c42cbcb59514a039fc73dd7abfa89183519f